a10_octavia/controller/worker/tasks/a10_network_tasks.py

```diff
@@ -817,6 +831,7 @@ def execute(self, vthunder, lb_resource, vrid_list, subnet,
             if vrid_subnet.id == subnet.id or vrid.vrid_floating_ip in existing_fips:
                 vrid_floating_ips.append(vrid.vrid_floating_ip)
 
+        self._create_vrid(vrid_value)
```

Comment:
In my opinion, we should not create the VRID here. We should do so in a different task. This way we can fail and revert if the VRID cannot be created for some reason.

We can just create a small `HandleVRID` task that runs before `HandleVRIDFloatingIP` and ensures the desired VRID exists or is created.

Reply:
done